cypress/integration/dib-travel/company-employees/employees.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { Employee } from '../../../models/employee/employee';
+
+describe('Company Employees - Employees Page', () => {
+  let employeeDetails: Employee;
+
+  before(() => {
+    cy.fixture('company-employees/employees').then((employee) => {
+      employeeDetails = employee;
+    });
+  });
+
+  beforeEach(() => {
```

Comment:
Your general use `beforeEach` should be something like:
```
cy.login();
cy.visit('/parent-feature/sub-feature');
```

And testing only sidebar link (and confirming loaded content) should be a separate single test.